diff --git a/node_modules/svelte-adapter-firebase/src/index.js b/node_modules/svelte-adapter-firebase/src/index.js
index 28b6478..ed67a89 100644
--- a/node_modules/svelte-adapter-firebase/src/index.js
+++ b/node_modules/svelte-adapter-firebase/src/index.js
@@ -24,7 +24,7 @@ const entrypoint = function (options = {}) {
 
 			builder.log.minor(`Adapter configuration:\n\t${JSON.stringify(options)}`);
 			const {functions, publicDir} = parseFirebaseConfiguration({firebaseJsonPath, target, sourceRewriteMatch});
-			ensureStaticResourceDirsDiffer({source: path.join(process.cwd(), builder.getStaticDirectory()), dest: publicDir});
+			ensureStaticResourceDirsDiffer({source: path.join(process.cwd(), builder.config.kit.files.assets), dest: publicDir});
 			const functionsPackageJson = JSON.parse(readFileSync(path.join(functions.source, 'package.json'), 'utf8'));
 			if (!functionsPackageJson?.main) {
 				throw new Error(`Error reading ${functionsPackageJson}. Required field "main" missing.`);
